crates/forge_main/src/model.rs

```diff
@@ -378,11 +378,18 @@ impl ForgeCommandManager {
                     // Handle custom workflow commands
                     let command_name = command.strip_prefix('/').unwrap_or(command);
                     if let Some(command) = self.find(command_name) {
-                        let value = self.extract_command_value(&command, &parts[1..]);
-
+                        let actual_command = self
+                            .extract_command_value(&command, &parts[1..])
+                            .unwrap_or_default();
+                        let parameters_str = parameters.join(" ");
```

Comment:
add some unit tests. Handle scernarios with newlines and multiple whitespaces/newlines.